_docs/tasks/basic-access-control.md

```diff
@@ -89,6 +101,25 @@ Using a whitelist is a two step process.
    `checkExpression` is evaluated and checked against the list `[v1, v2]`. The check behavior can be changed to a blacklist by specifying
    `blacklist: true`. The expression evaluator returns the value of the `version` label as specified by the `checkExpression` key.
 
+The alpha version of `istioctl` does not yet support
+pushing adapter configurations, although there is a
+[workaround]({{home}}/docs/concepts/policy-and-control/mixer-aspect-config.html#pushing-configuration)
+if you want to try it.
+
+## Cleanup
+
+* Remove the mixer configuration rule:
+
+  ```bash
+  istioctl mixer rule create global ratings.default.svc.cluster.local -f samples/apps/bookinfo/mixer-rule-empty-rule.yaml
```

Comment:
This seems strange doesn't it? Using a create with empty object. Not sure if we can use istioctl delete. If not, this is a thing that we can definitely fix in istioctl (@ayj @esnible)